db/fns.md

```diff
@@ -1794,6 +1799,16 @@ Return true if the task has remaining un-satisfied dependencies.
 
 temp, removed in next commit
```

Comment:
huh, this sounds like me.  whoops?

Reply:
thanks for spotting and bringing that back!